objcgen/objcgenerator.cs

```diff
@@ -371,6 +376,59 @@ protected override void Generate (PropertyInfo pi)
 			ImplementMethod (setter, "set" + pi.Name, pi);
 		}
 
+		protected void Generate (FieldInfo fi)
+		{
+			bool read_only = ((fi.Attributes & FieldAttributes.InitOnly) == FieldAttributes.InitOnly);
+
+			headers.Write ("@property (nonatomic");
+			if (fi.IsStatic)
+				headers.Write (", class");
+			if (read_only)
+				headers.Write (", readonly");
+			else
+				headers.Write (", readwrite");
```

Comment:
It's already generated for _normal_ properties - but we can PR their removal (both) since, being the default, it has no/low value to ObjC developers.